lxcfs 6.0.1 PATH_MAX undeclared under musl

# Issue description

lxcfs 6.0.1 build failed under musl system due to missing `PATH_MAX`. 
It should includes `limits.h`.

# Information to attach
```
clang -Iliblxcfs.so.p -I. -I../lxcfs-6.0.1 -I/usr/include/fuse3 -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -std=gnu11 -Wvla -Wcast-align -Wstrict-prototypes -fno-strict-aliasing -fstack-clash-protection -fstack-protector-strong --param=ssp-buffer-size=4 -Werror=implicit-function-declaration -Wmissing-include-dirs -Wold-style-definition -Winit-self -Wunused-but-set-variable -Wno-unused-parameter -Wfloat-equal -Werror=return-type -Werror=incompatible-pointer-types -Wformat=2 -Wshadow -Wendif-labels -Werror=overflow -fdiagnostics-show-option -Werror=shift-count-overflow -Wdate-time -Wnested-externs -fasynchronous-unwind-tables -fexceptions -Warray-bounds -Wreturn-local-addr -include config.h -O2 -pipe -fPIC -pthread -MD -MQ liblxcfs.so.p/src_utils.c.o -MF liblxcfs.so.p/src_utils.c.o.d -o liblxcfs.so.p/src_utils.c.o -c ../lxcfs-6.0.1/src/utils.c
In file included from ../lxcfs-6.0.1/src/utils.c:24:
../lxcfs-6.0.1/src/bindings.h:116:27: error: use of undeclared identifier 'PATH_MAX'
  116 |         char runtime_path[PATH_MAX];
      |                           ^
1 error generated.
```

--- a/src/bindings.h
+++ b/src/bindings.h
@@ -5,6 +5,7 @@
 
 #include "config.h"
 
+#include <linux/limits.h>
 #include <linux/types.h>
 #include <signal.h>
 #include <stdbool.h>

